serverless/messaging/reference-content/limitations.mdx

```diff
@@ -7,12 +7,14 @@ content:
   paragraph: Learn about the current limitations of Scaleway Messaging and Queuing
 tags: messaging limitations space size storage
 dates:
-  validation: 2023-07-10
+  validation: 2023-10-05
   posted: 2023-01-04
 categories:
   - serverless
 ---
 
+## Usage limits
+
```

Comment:
The table below references "namespace" many types. For NATS should it be changed to NATS account? For SQS and SNS - Scaleway Project?

Reply:
yes, I guess it would be:

```
| Resource                                       | Criteria   | Limits   | Scope        |
|------------------------------------------------|------------|----------|--------------|
| NATS account                                      | Max number | 10       | Organization |
| NATS message payload                           | Max size   | 1 MB     | Message      |
| SQS/SNS message payload                        | Max size   | 256 KB   | Message      |
| SQS queue storage                              | Max size   | 100 MB   | Scaleway Project    |
| NATS queue/stream storage                      | Max size   | 300 MB** | NATS account    |
| Max number of NATS/ streams                 | Max number | 10       | NATS account    |
| Max number of SQS queues                  | Max number | 10       | Scaleway Project    |
| Max number of SNS topics                  | Max number | 10       | Scaleway Project    |
| Max number of NATS consumers | Max number | 50      | NATS account    |
| Max number of SNS subscriptions | Max number | 50      | Scaleway Project    |
| Message age (SQS)                              | Max        | 14 days  | Message      |
```

Maybe it would be nicer to split this table for NATS vs SQS/SNS.